src/intTest/java/com/box/sdk/MetadataTemplateIT.java

```diff
@@ -267,6 +285,45 @@ public void getAllMetadataSucceeds() {
         }
     }
 
+    @Test
+    public void executeMetadataTemplateQuery() {
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+        String templateKey = "MyTemplate";
+        BoxFolder one = null;
+        BoxFolder two = null;
+
+        try {
+            MetadataTemplate.Field metadataField = new MetadataTemplate.Field();
+            metadataField.setType("float");
+            metadataField.setKey("myField");
+            metadataField.setDisplayName("Value");
+            List<MetadataTemplate.Field> fields = new ArrayList<>();
+            fields.add(metadataField);
+            MetadataTemplate.createMetadataTemplate(api, "enterprise", templateKey, "My Template", false, fields);
+
+            BoxFolder rootFolder = getUniqueFolder(api);
+            one = rootFolder.createFolder("one").getResource();
+            one.createMetadata(templateKey, "enterprise", new Metadata().add("/myField", 102d));
+            two = rootFolder.createFolder("two").getResource();
+            two.createMetadata(templateKey, "enterprise", new Metadata().add("/myField", 95d));
+
+
+            MetadataQuery query = new MetadataQuery("enterprise_867393134.MyTemplate")
```

Comment:
Can we somehow replace this ID either with value from config or fetched from server?